docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml

```diff
@@ -97,8 +97,7 @@ methods:
     fullName: getRange()
     summary: >-
       Returns the `Range` object that represents the range to which the AutoFilter applies. If there is no `Range`
-      object associated with the AutoFilter, then this method returns an object with its `isNullObject` property set to
-      `true`<!-- -->.
+      object associated with the AutoFilter, then this method returns `undefined`<!-- -->.
```

Comment:
This description uses "method", but all the others use "function". I recall you changing function to method at one point (in a different PR related to isNullObject). What are your thoughts on that variation here?

Reply:
It's an unintentional difference, but one that's in numerous other places in the docs. This Considering the amount of upstream effort for this one change, I'd rather fix the method/function distinction in the base metadata.